x/mongo/driver/serverapioptions.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package driver
+
+// ServerAPIOptions represents options used to configure the API version sent to the server
+// when running commands.
+type ServerAPIOptions struct {
+	ServerAPIVersion  string
+	Strict            *bool
+	DeprecationErrors *bool
+}
+
+// ServerAPI creates a new ServerAPIOptions configured with default values. ServerAPIVersion is set
+// to latest stable version.
+func ServerAPI() *ServerAPIOptions {
```

Comment:
Can we rename to `NewServerAPIOptions`? That's the general pattern constructors should follow. We called it `ServerAPI` for the user-facing options types because they're in the `options` package and it's a bit less verbose.

Reply:
Renamed to `NewServerAPIOptions`!